posthog/api/test/test_event.py

```diff
@@ -150,4 +150,84 @@ def test_before_and_after(self):
         
         response = self.client.get('/api/event/?before=2020-01-09').json()
         self.assertEqual(len(response['results']), 1)
-        self.assertEqual(response['results'][0]['id'], event2.pk)
\ No newline at end of file
+        self.assertEqual(response['results'][0]['id'], event2.pk)
+
+    def test_sessions_avg_length(self):
+        with freeze_time("2012-01-14 03:21:34"):
```

Comment:
For all of these, if you pass it as `"2012-01-14T03:21:34.000Z"` you don't get the timezone warnings